python/tests/test_utils.py

```diff
@@ -0,0 +1,69 @@
+from numpy.testing import assert_allclose
+
+import rascaline
+from rascaline.systems import IntoSystem
+
+
+def finite_differences_positions(
+    calculator: rascaline.CalculatorBase,
+    system: IntoSystem,
+    displacement: float = 1e-6,
+    rtol: float = 1e-5,
+    atol: float = 1e-6,
+) -> None:
+    """
+    Check that analytical gradients with respect to positions agree with a finite
+    difference calculation of the gradients.
+
+    The implementation is simular to ``rascaline/src/calculators/tests_utils.rs``.
+
+    :param calculator: calculator used to compute the representation
+    :param system: Atoms object
+    :param displacement: distance each atom will be displaced in each direction when
+        computing finite differences
+    :param max_relative: Maximal relative error. ``10 * displacement`` is a good
+        starting point
+    :param atol: Threshold below which all values are considered zero. This should be
+        very small (1e-16) to prevent false positives (if all values & gradients are
```

Comment:
The default value does not respect this doc comment =) Should we update it to be something like 1e-12 or 1e-16?